src/ducks/transactions/Transactions.jsx

```diff
@@ -129,35 +126,60 @@ const TableTrNoDesktop = translate()(
                   getCategoryId(transaction)
                 )}`
               )}
+              onClick={() => selectTransaction(transaction)}
             >
               <CategoryIcon
                 category={getParentCategory(getCategoryId(transaction))}
               />
             </Img>
             <Bd className="u-mr-half u-ellipsis">
-              <ListItemText
-                primaryText={getLabel(transaction)}
-                secondaryText={
-                  !filteringOnAccount && getAccountLabel(transaction.account)
-                }
-              />
+              <div onClick={() => selectTransaction(transaction)}>
+                <ListItemText
+                  primaryText={getLabel(transaction)}
+                  secondaryText={
+                    !filteringOnAccount && getAccountLabel(transaction.account)
+                  }
+                />
+              </div>
             </Bd>
-            <Img style={{ flexBasis: '1rem' }}>
-              <TransactionActions
-                transaction={transaction}
-                {...props}
-                onlyDefault
-                onlyIcon
-              />
-            </Img>
-            <Img>
+            <Img onClick={() => selectTransaction(transaction)}>
               <Figure
                 total={transaction.amount}
                 currency={transaction.currency}
                 coloredPositive
                 signed
               />
             </Img>
+            <Img
+              style={{
+                position: 'relative',
+                flexBasis: '2rem',
+                flexShrink: '0',
+                flexGrow: '0',
+                textAlign: 'right',
+                marginLeft: '1rem',
+                paddingLeft: '0.4rem'
+              }}
```

Comment:
can this be extracted either in a CSS file or outside the render function ?

Reply:
Yes.